pubspec.yaml

```diff
@@ -10,10 +10,10 @@ environment:
   sdk: ">=2.7.0 <3.0.0"
```

Comment:
You tested all this?

Reply:
Its working perfectly fine in production right now (Flutter Web, master branch)

Reply:
hmm... need retest in stable branch, within 30 minutes I will check

Reply:
need downgrade xml to 3.6.1

```
Because every version of flutter_test from sdk depends on xml 3.6.1 and every version of easy_localization_loader from git depends on xml ^4.2.0, flutter_test from sdk is incompatible with easy_localization_loader from git.
So, because example depends on both easy_localization_loader from git and flutter_test any from sdk, version solving failed.
```

Reply:
andd please change SDK depends to ```sdk: ">=2.7.0 <3.0.0"```

Reply:
I didn't change any SDK version, SDK was and is ">=2.7.0 <3.0.0"
Sorry I didnt check flutter_test compatibility with xml 4.0.0+ and sadly this whole commit was because one of our other dependencies relies on xml 4.0.0+. We would be using our fork of easy_localization_loader for now

Reply:
You can youse this package, but you need oride depency in pubspec

```
dependency_overrides:
  xml: "^4.2.0"
```

Reply:
Wow! Thanx, I didn't knew about it